packages/server/src/email/email.service.ts

```diff
@@ -5,22 +5,15 @@ import * as ses from 'node-ses';
 @Injectable()
 export class EmailService {
   private readonly client = ses.createClient({} as any);
-
+  
```

Comment:
NIT (short for nit pick): Best to not have white space changes. I've added a feature to my IDE to remove extra spaces/etc at the end of lines for things like that.